CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 **Details**
 
 - `nflow-engine`
+  - Improve shutdown sequence.
+    - Workflows that have been acquired from database, but had not yet started executing, are now resumed immediately by another executor.
+    - During the last 5 seconds of shutdown timeout the workflows are interrupted so that they persist their state and can be resumed immediately.
```

Comment:
```suggestion
    - Workflows that were acquired from the database but are not yet started executing can now be resumed immediately by another executor.
    - Executing workflows are interrupted 5 seconds before shutdown timeout so that they get a chance to persist their state to the database. This also allows other executors to immediately resume the processing of the successfully interrupted workflows.
```